site/_posts/2020-08-06-securely-implement-oauth-vuejs.md

```diff
@@ -2,9 +2,10 @@
 layout: blog-post
 title: How to Securely Implement OAuth in Vue.js
 description: This post describes how to securely implement OAuth in a Vue application using the Authorization Code Grant (with FusionAuth as the IdP).
-author: Ashutosh Singh 
-image: blogs/oauth-vuejs/how-to-securely-implement-oauth-in-vue-js.png     
+author: Ashutosh Singh
+image: blogs/oauth-vuejs/how-to-securely-implement-oauth-in-vue-js.png
 category: blog
+updated_date: 2021-03-16
 tags: client-javascript
```

Comment:
We discussed using tags here to mark tutorials so we could build a page similar to the example apps page, but for tutorials.

Can you add the relevant tutorial tags here, please?